multicluster/cmd/service-mirror/main.go

```diff
@@ -288,8 +288,11 @@ func restartClusterWatcher(
 	if err != nil {
 		return fmt.Errorf("failed to create metrics for cluster watcher: %w", err)
 	}
-	probeWorker = servicemirror.NewProbeWorker(fmt.Sprintf("probe-gateway-%s", link.TargetClusterName), &link.ProbeSpec, workerMetrics, link.TargetClusterName)
-	probeWorker.Start()
+
+	if link.ProbeSpec.Path != "" {
+		probeWorker = servicemirror.NewProbeWorker(fmt.Sprintf("probe-gateway-%s", link.TargetClusterName), &link.ProbeSpec, workerMetrics, link.TargetClusterName)
```

Comment:
`ProbeWorker` remains nil in this case, which creates a panic below when referencing `probeWorker.Liveness`

Reply:
Good spot! I fixed this, controlled started with both a Link w/o a probeSpec, and a Link w/ a probeSpec.